corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -89,7 +89,95 @@ var ReportModule = (function () {
         this.allGraphTypes = ['bar', 'time', 'xy'];
     }
 
-    function ReportConfig(report_id, display, availableReportIds, reportCharts, graph_configs, language) {
+    function FilterConfig(report_id, reportId, filterValues, reportFilters, changeSaveButton) {
```

Comment:
`report_id` vs `reportId` is not a great way to distinguish between two variables